hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -192,30 +192,28 @@ The [HDMI boot diagnostics](./boot_diagnostics.md) display is disabled if DISABL
 Default: 0   
 Version: pieeprom-2020-04-16.bin  
 
-`Version: pieeprom-2020-06-15.bin - STABLE`
+`Version: pieeprom-2020-09-03`
 The `disable_splash` property is no longer checked because the HDMI diagnostics screen is started before config.txt is read.
 
 ### ENABLE_SELF_UPDATE
-Allows the bootloader to update itself instead of requiring recovery.bin. This is intended to make it easier to update the bootloader firmware via network boot. To enable set `ENABLE_SELF_UPDATE=1` and add `bootloader_update=1` in config.txt.
-N.B. There is no automatic rollback in the event of a power failure during the firmware update. In the unlikely event of this happening you will have to use Pi Imager to apply the rescue image.
+Enables the bootloader to update itself from a network or USB-MSD boot.
 
-If self update is enabled then the bootloader will look for (pieeprom.upd + pieeprom.sig) and/or (vl805.bin + vl805.sig) on the boot partition (or TFTP root). If the update files are different to the current image then the update is applied and system is reset. Otherwise, if the images are identical then boot continues as normal.
+If self update is enabled then the bootloader will look for (pieeprom.upd + pieeprom.sig) and/or (vl805.bin + vl805.sig) on the USB MSD boot partition (or TFTP root). If the update files are different to the current image then the update is applied and system is reset. Otherwise, if the images are identical then boot continues as normal.
 
 Since the updates files are in the same format as generated by rpi-eeprom-update you can use rpi-eeprom-update to install the files to /boot so long as /boot mounts the approprate boot device / tftp-root (via NFS).
 
-**The bootloader only reads the configuration from the bootconf.txt in the EEPROM and not the one in the update files. Therefore, in order to enable `ENABLE_SELF_UPDATE` you have to first update the bootloader via the SD-CARD or FLASHROM.**
+Notes:-
+*  Self-update is not enabled in SD boot because the ROM can already load recovery.bin from the SD card. 
+*  Before self-update can be used (e.g. from USB boot) the bootloader must have already been updated to a version which supports self update. The recommended approach would be to use the Raspberry Pi Imager and a spare SD card to update to pieeprom-2020-09-03 then use self-update for subsequent updates.
 
-Default: 0  
-Version: pieeprom-2020-04-16.bin  
-
-`Version: pieeprom-2020-06-15.bin - STABLE`
-The `ENABLE_SELF_UPDATE` EEPROM property and `bootloader_update` config.txt property are now enabled by default so that `rpi-eeprom-update` may be used without requiring extra customization. Setting either of these parameters to zero prevents self-updates.
+Default: 1  (0 in earlier releases)
+Version: pieeprom-2020-09-03.bin  
```

Comment:
The history of this file seems to imply that ENABLE_SELF_UPDATE was added in pieeprom-2020-04-16.bin, and pieeprom-2020-09-03.bin only changed the default value from 0 to 1? (but it's obviously up to you how "accurate" you want to keep these version-availability tags)

Reply:
Added a note since 2020-04-16 was a production release - I think it's reasonable for the epoch for a feature to be the first production release.

Reply:
reemoved pieeprom- and .bin from all versions. Just use the date